doc/gitian-building.md

```diff
@@ -336,6 +336,8 @@ There will be a lot of warnings printed during the build of the image. These can
 Getting and building the inputs
 --------------------------------
 
+At this point you have two options, you can either use the automated script (found in [contrib/gitian-build.sh](/contrib/gitian-build.sh)) or you could manually do everything by following this guide.
+
```

Comment:
Concept ACK, although it would be nice to specify here how to use the script (eg, what to type to build, like the rest of the document) instead of just referring to it.